integration_tests/pytest.ini

```diff
@@ -36,5 +36,6 @@ markers =
     large_data_test: Mark tests with large data
     pyarrow_test: Mark pyarrow tests
     datagen_overrides: Mark that allows overriding datagen settings (i.e. seed) for a test
+    tz_sensitive_test: Mark the test as a time zone sensitive test which will be tested against extra timezones
```

Comment:
maybe: test_in_premerge_CI_with_non_UTC_TZ
Or other meaningful name.

Reply:
IMHO, we don't need to add such a long tag because the annotation for timezone sensitive test cases might be used for other purposes in future.